packetnetworking/distros/debian/dhcp.py

```diff
@@ -0,0 +1,31 @@
+from .. import NetworkBuilder
+from ...utils import generate_persistent_names
+
+
+# pylama:ignore=E501
+class DebianDhcpNetwork(NetworkBuilder):
+    def build(self):
+        if self.network.bonding.link_aggregation == "individual":
+            self.build_tasks()
+
+    def build_tasks(self):
+        self.tasks = {}
+
+        self.task_template(
+            "etc/network/interfaces", "dhcp/etc_network_interfaces.j2"
+        )
+
+        os = self.metadata.operating_system
+
+        if os.distro == "debian" and os.version in ["10", "11"]:
+            self.tasks.update(generate_persistent_names())
+        elif os.distro == "ubuntu" and os.version in [
```

Comment:
In Debian family, we only support ubuntu 18.04 and 20.04. For Q4, we agreed with CCP only support 20.04